Services/data/devicemodels/truck-01-protobuf.json

```diff
@@ -2,7 +2,7 @@
     "SchemaVersion": "1.0.0",
     "Id": "truck-01-proto",
     "Version": "0.0.1",
-    "Name": "Truck 01 Proto",
+    "Name": "Protobuf Truck",
```

Comment:
nit: I'd rename this to "Truck sending telemetry in Protobuf format" or something similar